config/initializers/content_api.rb

```diff
@@ -12,6 +12,16 @@ def tags(tag_type, options = {})
   def artefact(*args)
     nil
   end
+
+  def artefacts_tagged_to_mainstream_browse_pages
+    []
+  end
+end
+
+class GdsApi::ContentApi
+  def artefacts_tagged_to_mainstream_browse_pages
+    get_json("#{base_url}/whitehall-artefacts-tagged-to-mainstream-browse-pages.json")
```

Comment:
This should probably be `get_json!` as we want to raise an error if the endpoint 404s.